chore: improve README of pipelines (#340)
kubevirt/kubevirt-tekton-tasks: release/pipelines/windows-bios-installer/README.md

1. `create-vm-root-disk` Task creates an empty DataVolume.
2. `create-vm` Task creates a VirtualMachine called `windows-bios-installer-*`
   from the empty DataVolume and with the `windows-bios-installer-cd-rom` DataVolume attached as a CD-ROM.

>    A second DataVolume with the virtio-win ISO will also be attached (Pipeline parameter `virtioContainerDiskName`). The VirtualMachine has to be created at the same namespace as the empty DataVolume.

Reviewer: ```suggestion
   A second DataVolume with the virtio-win ISO will also be attached (Pipeline parameter `virtioContainerDiskName`). The VirtualMachine has to be created in the same namespace as the empty DataVolume.
```

3. `wait-for-vmi-status` Task waits until the VirtualMachine shuts down.
4. `cleanup-vm` deletes the installer VirtualMachine and ISO DataVolume (also in case of failure of the previous Tasks).
5. The output artifact will be the `win10` DataVolume with the basic Windows installation.
   It will boot into the Windows OOBE and needs to be setup further before it can be used.

## How to run

> Before running PipelineRuns, make sure, the ConfigMaps with autounattend.xml exists in the same namespace as VirtualMachine will be created!

Reviewer: ```suggestion
Before you create PipelineRuns, you must create ConfigMaps with an autounattend.xml in the same namespace in which the VirtualMachine will be created.
```
